docs/resources/mte_config.md

```diff
@@ -62,6 +63,7 @@ Optional:
 
 - `append_path_prefix` (String) A string which will be appended to the start of the path sent to the host.
 - `cache_key` (Attributes) An object specifying header and cookie names which should be added to the cache key. The result of this would lead to separate cache hits for requests with different values of the header or cookie. (see [below for nested schema](#nestedatt--config--routes--cache_key))
+- `cache_max_age` (String) A string of an int that will be used to specify the time that the response of the route should be stored in the cache, in seconds.
```

Comment:
Why don't we just make this an integer?

Reply:
We could do, the reason I didn't initially was to keep it more consistent with what we use downstream. Happy to change, i imagine it will need some changes in the other repos.

Reply:
I think we need to add this to the API anyway so we control what should be passed and accepted. I would recommend this being an integer though